x-pack/plugins/observability_solution/apm/public/components/app/service_tab_content/service_tab_content.stories.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ComponentProps, ComponentType } from 'react';
+import { ServiceTabContent } from '.';
+import {
+  ApmPluginContext,
+  ApmPluginContextValue,
+} from '../../../context/apm_plugin/apm_plugin_context';
+
+const contextMock = {
+  core: { http: { basePath: { prepend: () => {} } } },
+} as unknown as ApmPluginContextValue;
+
+export default {
+  title: 'APP/ServiceTabContent',
```

Comment:
Thanks for creating a storybook 👏🏻

Reply:
I used it at the start of the feature with the isOnlyLogs flag hardcoded to true, and I see that it’s now misconfigured and not working properly. I’ll look into updating it to make it more functional.